### PR TITLE
Upgrade LiT to v0.6.2 (includes bugfix)

### PR DESCRIPTION
**Pull request by dennisreimann**
Version bump (closes btcpayserver/btcpayserver#3459) and two bugfixes:

- Persist the `LIT_PASSWD` in `.env`
- ac398d569f75a3fb6c9889b1e0f1d75e93725a1c fixes btcpayserver/btcpayserver-docker#601

### CODE REVIEW

File: docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     environment:
       BTCPAY_EXTERNALSERVICES: "Lightning Terminal:${BTCPAY_PROTOCOL:-https}://${BTCPAY_HOST}/lit/;"
   lnd_lit:
-    image: "lightninglabs/lightning-terminal:v0.6.1-alpha-path-prefix"
+    image: "lightninglabs/lightning-terminal:v0.6.2-alpha-path-prefix"
```
Comment (guggero) [inline docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml:7]: If we're updating this file, shouldn't we also make sure that all of LiT's data is written to the volume correctly? I see there is a volume at `/data/lit`, but LiT isn't configured to use it. Either an ENV var `HOME=/data/lit` should be set (or is this done somewhere else?) or the volume should be mounted to `/root`. See https://github.com/btcpayserver/btcpayserver-docker/issues/592. Currently Pool's data (and I assume Loop, Faraday and Lit's internal data) is lost when the container is re-created.

File: Production/nginx.tmpl [hunk 2]
```diff
@@ -144,15 +144,7 @@
...
-	location ~* ^/lnrpc.Lightning/ {
-    proxy_pass http://lnd_lit:8080;
-  }
-
-  location ~* ^/looprpc.SwapClient/ {
-    proxy_pass http://lnd_lit:8080;
-  }
-
-  location ~* ^/poolrpc.Trader/ {
+  location ~* ^/(ln|loop|pool|lit)rpc\. {
```
Comment (guggero) [inline Production/nginx.tmpl:147]: :tada:

**Review by guggero — COMMENTED** [review]
Thanks for the update! Changes look good to me, but I think it would be great if we could also address the data/volume issue (see inline comment).

File: docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml (hunk 1 above)
Comment (dennisreimann) [reply docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml:7]: Thanks for the pointer - ok like [cfaed5f](https://github.com/btcpayserver/btcpayserver-docker/pull/602/commits/cfaed5fa23c0ae384b20c5c661c781e7fde6d175)?

Comment (guggero) [reply docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml:7]: Looks good, yes. Thanks! I'm not able to test this though.

**Pull request merged**